javascript-challenges.test.js

```diff
@@ -22,12 +30,35 @@ const secretCodeWord3 = "Eccentric"
 // Expected output: "3cc3ntr1c"
 
 // b) Create the function that makes the test pass.
+const stringToCode = (String) => {
+    const lettersToNum = {'a': '4','e': '3','i': '1', 'o': '0'}
+  
+ return String.split('').map(arr => lettersToNum[arr.toLowerCase()] || arr).join('')
```

Comment:
Excellent work on this problem!  One thing to keep an eye on is the casing and naming conventions used.  Usually a parameter would be in camelCase in JS, and if you are mapping over an array of letters, the parameter name would usually correlate with the data (in this case a letter).